zerver/tests/test_message_edit.py

```diff
@@ -4476,7 +4476,7 @@ def test_delete_event_sent_after_transaction_commits(self) -> None:
         message = self.get_last_message()
 
         with self.capture_send_event_calls(expected_num_events=1):
-            with mock.patch("zerver.actions.message_edit.send_event") as m:
```

Comment:
(This should have been adjusted to `zerver.actions.message_delete.send_event` in 4303ba1efc0fd3325cb60097819fc7535d8785b0 from #22506.)